src/pages/[id].page.jsx

```diff
@@ -13,9 +13,12 @@ export default function Details() {
   );
 }
 
+export const getStaticPaths = async () => ({
+  paths: [], // indicates that no page needs be created at build time
```

Comment:
I think we actually will know what paths we need in advance. We can put in the list of components.

```suggestion
  paths: [
    '/accordion',
    '/dialog',
    '/disclosure',
    '/tabs',
  ],
```